tests/iterators.rs

```diff
@@ -544,9 +543,9 @@ fn axis_chunks_iter_corner_cases() {
     assert_equal(
         it,
         vec![
-            arr2(&[[7.], [6.], [5.]]),
-            arr2(&[[4.], [3.], [2.]]),
-            arr2(&[[1.], [0.]]),
```

Comment:
what was the warning here?

Reply:
Unused import of `arr2` when `std` feature is not enabled.